client/components/main/index.jsx

```diff
@@ -11,7 +11,11 @@ module.exports = React.createClass( {
 
 	render: function() {
 		return (
-			<main className={ classnames( this.props.className, 'main' ) } role="main">
+			<main
+				className={ classnames( this.props.className, 'main' ) }
+				role="main"
+				style={ this.props.style }
```

Comment:
Hmm, odd that we need to override style here. Is there a specific use case that we couldn't pass through a css class name for?